src/mame/f256/tiny_vicky.h

```diff
@@ -0,0 +1,81 @@
+// license:BSD-3-Clause
+// copyright-holders:Daniel Tremblay
+#ifndef MAME_F256_TINY_VICKY_H
+#define MAME_F256_TINY_VICKY_H
+
+#define CHAR_HEIGHT     8
+#define CHAR_WIDTH      8
+
+//#include <atomic>
+
+class tiny_vicky_video_device : public device_t
+{
+public:
+    tiny_vicky_video_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock);
+    tiny_vicky_video_device(const machine_config &mconfig, device_type type, const char *tag, device_t *owner, uint32_t clock = 0);
+    //tiny_vicky_video_device() = default;
+
+    // screen update
+    uint32_t screen_update(screen_device &screen, bitmap_rgb32 &bitmap, const rectangle &cliprect);
+    void stop()
+    {
+        running = false;
+        videoram_ptr = nullptr;
+        iopage0_ptr = nullptr;
+        iopage1_ptr = nullptr;
+        iopage2_ptr = nullptr;
+        iopage3_ptr = nullptr;
+    }
+    void set_videoram(uint8_t *videoram, uint8_t *iopage0, uint8_t *iopage1, uint8_t *iopage2, uint8_t *iopage3)
+    {
+        videoram_ptr = videoram;
+        iopage0_ptr = iopage0;
+        iopage1_ptr = iopage1;
+        iopage2_ptr = iopage2;
+        iopage3_ptr = iopage3;
+    }
+    void start()
+    {
+        running = true;
+    };
+    auto sof_irq_handler()
+    {
+        return m_sof_irq_handler.bind();
+    };
+    auto sol_irq_handler()
+    {
+        return m_sol_irq_handler.bind();
+    };
+protected:
+    // device-level overrides
+	virtual void device_start() override ATTR_COLD;
+	virtual void device_reset() override ATTR_COLD;
+    // virtual void device_add_mconfig(machine_config &config) override;
+private:
+    bool running = false;
```

Comment:
Class variables should have `m_` prefix.

Reply:
I've modified all the private class variables to have the m_ prefix.